<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mumble Server Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="qtwServers">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ping</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Users</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="qdbbButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="qaFavoriteRemove">
   <property name="text">
    <string>Remove from Favorites</string>
   </property>
  </action>
  <action name="qaFavoriteEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="qaFavoriteAddNew">
   <property name="text">
    <string>Add New...</string>
   </property>
   <property name="toolTip">
    <string>Add custom server</string>
   </property>
  </action>
  <action name="qaFavoriteAdd">
   <property name="text">
    <string>Add to Favorites</string>
   </property>
  </action>
  <action name="qaUrl">
   <property name="text">
    <string>Open Webpage</string>
   </property>
  </action>
  <action name="qaHideUnreachable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide unreachable</string>
   </property>
   <property name="toolTip">
    <string>Hide servers that don't reply to ping</string>
   </property>
  </action>
  <action name="qaHideEmpty">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide empty</string>
   </property>
   <property name="toolTip">
    <string>Hide servers with no users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qdbbButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qdbbButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
